contracts/interfaces/IRolesBase.sol

```diff
@@ -15,8 +15,8 @@ interface IRolesBase {
     error InvalidProposedRoles(address fromAccount, address toAccount, uint8[] roles);
 
     event RolesProposed(address indexed fromAccount, address indexed toAccount, uint8[] roles);
-    event RolesAdded(address indexed account, uint8[] roles);
-    event RolesRemoved(address indexed account, uint8[] roles);
+    event RolesAdded(address indexed caller, address indexed account, uint8[] roles);
```

Comment:
What do we need this for? The way _addRoles etc. would be implemented usually would be to use msg.sender as the account itself. And in some instances, msg.sender can be ITS contract. The pattern of adding the caller to events feels weird since it could be applied to all events.